cleanlab/datalab/internal/issue_manager/utils.py

```diff
@@ -0,0 +1,28 @@
+from typing import Dict, Union, Optional, Any
```

Comment:
It might be beneficial to add a file named `test_utils.py` in `tests/datalab/issue_manager` and write basic tests for `process_knn_graph_from_inputs`.